packages/ui/molecules/Hero/Hero.twig

```diff
@@ -0,0 +1,143 @@
+ *   customize the image element attributes.
+ * @param array                                     $content_attr
+ *   customize the content classes.
+ * @param 'left'|'center'|'right'                   $content_position
+ *   define the content position.
+ * @param array                                     $surtitle_tag
+ *   customize the tag of the surtitle element
+ * @param array                                     $surtitle_attr
+ *   customize the surtitle element attributes.
+ * @param array                                     $title_tag
+ *   customize the tag of the title element
+ * @param array                                     $title_attr
+ *   customize the title element attributes.
+ * @param array                                     $description_tag
+ *   customize the tag of the description element
+ * @param array                                     $description_attr
+ *   customize the description element attributes.
+ * @param array                                     $cta_attr
+ *   customize the cta element attributes.
+ */
+#}
+
+{% set attributes =
+  merge_html_attributes(attr ?? null, { class: ['hero', 'w-screen', 'h-screen', 'relative'] })
+%}
+
+{# Default images attributes #}
+{% set image_attributes =
+  merge_html_attributes(
+    image_attr ?? null,
+    {
+      class: ['hero__image', 'object-cover', 'w-full', 'h-full', 'absolute', 'z-under'],
+      src: src,
+      alt: alt|default(''),
+      width: width|default(false),
+      height: height|default(false)
+    }
+  )
+%}
+
+{# Default content attributes #}
+{% set content_position = content_position|default('center') %}
+{% set content_attributes =
+  merge_html_attributes(
+    content_attr ?? null,
+    { class: 'hero-content' },
+    {
+      class: [
+        'flex flex-col justify-center h-full w-full space-y-8',
+        {
+          'items-start': content_position == 'left',
+          'items-center': content_position == 'center',
+          'items-end': content_position == 'right'
+        }
+      ]
+    }
+  )
+%}
+{% set surtitle_tag = title_tag|default('p') %}
+{% set surtitle_attributes =
+  merge_html_attributes(title_attr ?? null, { class: ['hero-content__surtitle', 'type-base'] })
+%}
+
+{% set title_tag = title_tag|default('p') %}
+{% set title_attributes =
+  merge_html_attributes(title_attr ?? null, { class: ['hero-content__title', 'type-h1'] })
+%}
+
+{% set description_tag = description_tag|default('p') %}
+{% set description_attributes =
+  merge_html_attributes(
+    description_attr ?? null,
+    { class: ['hero-content__description', 'type-base'] }
+  )
+%}
+
+{% set cta_attributes =
+  merge_html_attributes(cta_attr ?? null, { class: ['hero-content__cta', 'flex', 'px-6'] })
+%}
+
+<section {{ html_attributes(attributes) }}>
+  {% block background %}
+    {% if image %}
+      <div {{ html_attributes(image_attributes) }}>
+        {% include '@ui/atoms/Figure/Figure.twig' with image only %}
```

Comment:
**suggestion:** I think we can force the `Figure.twig` component in `absolute` here

```suggestion
        {% include '@ui/atoms/Figure/Figure.twig' with image|merge({ absolute: true }) only %}
```

This will ensure that it takes all the space available in its parent container.